mach

```diff
@@ -14,11 +14,10 @@ import sys
 
 # Destructure because version_info > max_ver is true when running the same version.
 ver = (sys.version_info[0], sys.version_info[1])
-min_ver = (3, 10)
-max_ver = (3, 12) # WPT does not support Python 3.13. See issue #34095.
-if ver < min_ver or ver > max_ver:
-    print("mach does not support python {0}.{1}, please install 3.{2} <= python <= 3.{3}" \
-        .format(ver[0], ver[1], min_ver[1], max_ver[1]))
+min_ver = (3, 9)
+if ver < min_ver:
```

Comment:
I noticed we have a similar [check in `mach_bootstrap.py` ](https://github.com/servo/servo/blob/caf629f7e14b734e0ef4fa59afcf807e7d31e50a/python/mach_bootstrap.py#L230). Is it redundant or am I missing something? Also, I just the minimum host version here to 3.9 as we no longer require host version to be too recent and 3.9 seemed like a good choice (5th most recent minor version). But maybe I should leave this at 3.10?

Reply:
@mrobinson do you have any thoughts here? Shall I remove the check in `mach_bootstrap.py` and leave the version here as 3.10?

Reply:
Do we still need this check if we are always guaranteed to be using the same version via `uv`?

Reply:
The check is for the host version of Python which is not guaranteed. We are probably only constrained by the minimum version required by `mach` itself and any servo code that mach's bootstrap process loads (I believe it dynamically loads the modules to get the commands).

Reply:
I think it's fine to remove the maximum version check and to remove the redundant check in `mach_bootstrap.py` if you think that's appropriate.